### PR TITLE
Add gridlayout

### PR DESCRIPTION
**Pull request by Raymundogs**
Delete unnecesary menu.java class, change interface name with gameboard and add a gridlayout

### CODE REVIEW

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java [hunk 1]
```diff
@@ -6,22 +6,22 @@
 import java.awt.event.KeyEvent;
 import java.awt.event.KeyListener;
 
-public class Interface extends JFrame {
+public class Gameboard extends JFrame {
```
Comment (JessickaMoya) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java:9]: Game board is two words. The class name should be GameBoard

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java [hunk 2]
```diff
@@ -6,22 +6,22 @@
...
-    private static final int POS_X = (Space.DEFAULT_WIDTH * 5 - 20) / 2;
-    private static final int POS_Y = Space.DEFAULT_HEIGHT * 14 - 20;
-    private static final int SCALE_WIDTH = Space.DEFAULT_WIDTH * 5;
-    private static final int SCALE_HEIGHT = Space.DEFAULT_HEIGHT * 15;
+    private static final int SIZE_IMAGE = 40;
+    private static final int POS_X = (Space.DEFAULT_WIDTH * 5 - SIZE_IMAGE) / 2;
+    private static final int POS_Y = Space.DEFAULT_HEIGHT * 14 - SIZE_IMAGE;
+    private static final int SCALE_WIDTH = Space.DEFAULT_WIDTH;
+    private static final int SCALE_HEIGHT = Space.DEFAULT_HEIGHT;
     private final Spaceship spaceship = new Spaceship((SCALE_WIDTH - SIZE_IMAGE) / 2, SCALE_HEIGHT - SIZE_IMAGE, 0, SCALE_WIDTH - 23);
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java:21]: If it's not using, should be commented for now, or erased.

**Review by JessickaMoya — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java [hunk 3]
```diff
@@ -6,22 +6,22 @@
 import java.awt.event.KeyEvent;
 import java.awt.event.KeyListener;
 
-public class Interface extends JFrame {
+public class Gameboard extends JFrame {
     private final JFrame frame = new JFrame();
     private final JPanel panel = new JPanel();
 
     private static JLabel ship = new JLabel(new ImageIcon("resources/spaceship.png"));
     private static final int GO_LEFT = 37;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java:14]: This is not used

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java [hunk 4]
```diff
@@ -6,22 +6,22 @@
...
     private static final int GO_RIGHT = 39;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Gameboard.java:15]: This is not used

**Review by danielaSantaCruz — APPROVED** [review]

**Pull request merged**